python_docs_theme/static/pydoctheme.css

```diff
@@ -129,6 +129,7 @@ form.inline-search input[type='submit'] {
 }
 
 div.document {
+    animation: fadeIn 0.6s ease-in-out;
```

Comment:
I would want to remove this, I didn't notice the document fade-in on mobile but having tested on my computer today it is noticeable.